iterfzf/__init__.py

```diff
@@ -32,6 +32,7 @@ def iterfzf(
     multi=False,
     mouse=True,
     print_query=False,
+    cycle=True,
```

Comment:
I haven't been aware of this option!  Thank you for letting me know.  Could you write docs about this option in *README.md*?  There is the section which deals with `iterfzf.itefzf()`'s parameters.